TinaDevLabApp/TinaDevLabApp/SpendTableViewCell.swift

```diff
@@ -7,17 +7,111 @@
+        view.addSubview(deleteButton)
+        
+        return view
+    }()
+    
+    //지출금액
+    public lazy var spendOutputLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.textColor = .black
+        
+        return label
+    }()
+    
+    //원
+    private lazy var spendUnitLabel: UILabel = {
+        let label = UILabel()
+        label.text = "원"
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.textColor = .black
+        
+        return label
+    }()
+    
+    //X버튼
+    private lazy var deleteButton: UIButton = {
+        let button = UIButton(type: .custom)
+//        button.setTitle("X", for: .normal)
+        button.setImage(UIImage(systemName: "x.circle"), for: .normal)
+        button.setTitleColor(UIColor.black, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+        button.addTarget(self, action: #selector(deleteButtonTapped), for: .touchUpInside)
+        
+        return button
+    }()
+    
+    // 초기화 메서드 - 스타일과 재사용 식별자를 매개변수로 받아 초기화를 수행
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        layoutCellView()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // Nib 파일로부터 객체가 초기화된 후 호출되는 메서드
     override func awakeFromNib() {
         super.awakeFromNib()
-        // Initialization code
     }
-
+    
+    // 셀의 선택 상태가 변경될 때 호출되는 메서드
     override func setSelected(_ selected: Bool, animated: Bool) {
         super.setSelected(selected, animated: animated)
-
-        // Configure the view for the selected state
     }
 
+    
+    // 공통 초기화 작업을 수행하는 메서드
+    private func layoutCellView() {
+        contentView.addSubview(cellBackground)
+        
+        cellBackground.translatesAutoresizingMaskIntoConstraints = false
+        spendOutputLabel.translatesAutoresizingMaskIntoConstraints = false
+        spendUnitLabel.translatesAutoresizingMaskIntoConstraints = false
+        deleteButton.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            cellBackground.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 25),
+            cellBackground.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -25),
+            cellBackground.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 11),
+            cellBackground.heightAnchor.constraint(equalToConstant: 50),
+            
+            spendUnitLabel.centerYAnchor.constraint(equalTo: cellBackground.centerYAnchor),
+            spendUnitLabel.trailingAnchor.constraint(equalTo: cellBackground.trailingAnchor, constant: -20),
+            
+            spendOutputLabel.leadingAnchor.constraint(equalTo: cellBackground.leadingAnchor, constant: 8),
```

Comment:
```suggestion
spendOutputLabel.trailingAnchor.constraint(equalTo: spendUnitLabel.trailingAnchor, constant: -15),
```

<img width="1416" alt="image" src="https://github.com/user-attachments/assets/26439531-9cb4-4245-bd34-89c359aabbde">

Reply:
<img width="223" alt="image" src="https://github.com/user-attachments/assets/06439bd2-635e-491b-9d83-3739b520a825">

엇 잘못 썼다
cellBackground.leading !!

Reply:
오 !!!!!! 라벨과 원의 관계를 고려하지 못했네 !!!!!!!!!!!!!

라벨의 오른쪽과 원의 왼쪽에 패딩을 준다고 이해했는데,
그렇다면 아래 코드처엄 spendUnitLabel.leadingAnchor 로도 줄 수 있을까?!

```swift
// 보노 코드
spendOutputLabel.trailingAnchor.constraint(equalTo: spendUnitLabel.trailingAnchor, constant: -15)

// 티나 생각
spendOutputLabel.trailingAnchor.constraint(equalTo: spendUnitLabel.leadingAnchor, constant: -15)

```